gsoft-inc/sg-orbit
# Milestones 5/dropdown

## Issues:
https://github.com/gsoft-inc/sg-orbit/issues/230
https://github.com/gsoft-inc/sg-orbit/issues/241
https://github.com/gsoft-inc/sg-orbit/issues/242
https://github.com/gsoft-inc/sg-orbit/issues/245

## What I did

- Exported the dropdown arrow svg in order to mitigate it's blurry appearance.
- Added support for non selection dropdown.
- Added support for errors in inline dropdowns.
- Added support for different font-size in inline dropdowns.
- Added support multiple values for select components.

## How to test

- Is this testable with Jest or Chromatic screenshots? **Chromatic** yes
- Does this need an update to the documentation? Could benefit from it.

If your answer is yes to any of these, please make sure to include it in your PR.
